wgpu/examples/framework.rs

```diff
@@ -1,6 +1,7 @@
 use std::future::Future;
 #[cfg(not(target_arch = "wasm32"))]
 use std::time::{Duration, Instant};
+use wgpu::DownlevelFlags;
```

Comment:
the rest of the code here refers to `wgpu` namespace explicitly, let's have this following